awx_collection/plugins/modules/tower_workflow_job_template.py

```diff
@@ -0,0 +1,209 @@
+        - Name of this workflow job template.
+      required: True
+      type: str
+    new_name:
+      description:
+        - Setting this option will change the existing name (looked up via the name field.
+      required: False
+      type: str
+    description:
+      description:
+        - Optional description of this workflow job template.
+      required: False
+      type: str
+    extra_vars:
+      description:
+        - NO DESCRIPTION GIVEN IN THE TOWER API
+      required: False
+      type: dict
+    organization:
+      description:
+        - Organization the workflow job template exists in.
+        - Used to help lookup the object, cannot be modified using this module.
+        - If not provided, will lookup by name only, which does not work with duplicates.
+      required: False
+      type: str
+    survey_enabled:
+      description:
+        - Enable a survey on the workflow job template.
+      required: False
+      type: bool
+    allow_simultaneous:
+      description:
+        - Allow simultaneous runs of the workflow job template.
+      required: False
+      type: bool
+    ask_variables_on_launch:
+      description:
+        - Prompt user for (extra_vars) on launch.
+      required: False
+      type: bool
+    inventory:
+      description:
+        - Inventory applied as a prompt, assuming job template prompts for inventory
+      required: False
+      type: str
+    limit:
+      description:
+        - Limit applied as a prompt, assuming job template prompts for limit
+      required: False
+      type: str
+    scm_branch:
+      description:
+        - SCM branch applied as a prompt, assuming job template prompts for SCM branch
+      required: False
+      type: str
+    ask_inventory_on_launch:
+      description:
+        - Prompt user for inventory on launch of this workflow job template
+      required: False
+      type: bool
+    ask_scm_branch_on_launch:
+      description:
+        - Prompt user for SCM branch on launch of this workflow job template
+      required: False
+      type: bool
+    ask_limit_on_launch:
+      description:
+        - Prompt user for limit on launch of this workflow job template
+      required: False
+      type: bool
+    webhook_service:
+      description:
+        - Service that webhook requests will be accepted from
+      required: False
+      type: str
+      choices:
+        - github
+        - gitlab
+    webhook_credential:
+      description:
+        - Personal Access Token for posting back the status to the service API
+      required: False
+      type: str
+    state:
+      description:
+        - Desired state of the resource.
+      choices:
+        - present
+        - absent
+      default: "present"
+      type: str
+    tower_oauthtoken:
+      description:
+        - The Tower OAuth token to use.
+      required: False
+      type: str
+      version_added: "3.7"
+extends_documentation_fragment: awx.awx.auth
+'''
+
+EXAMPLES = '''
```

Comment:
We need to add some examples here, but could do it in a separate PR

Reply:
That's a good point, this needs something filled in.